ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh

```diff
@@ -214,4 +214,4 @@ check_marketplace
 create_catalog_sources
 
 #support hypershift config guest cluster's icsp
-oc get imagecontentsourcepolicy -oyaml > /tmp/mgmt_iscp.yaml && yq-go r /tmp/mgmt_iscp.yaml 'items[*].spec.repositoryDigestMirrors' -  | sed  '/---*/d' > ${SHARED_DIR}/mgmt_iscp.yaml
\ No newline at end of file
+oc get imagecontentsourcepolicy -ojsonpath='{range .items[*].spec.repositoryDigestMirrors[*]}- mirrors:{"\n"}  - {@.mirrors[0]}{"\n"}  source: {@.source}{"\n"}{end}' > ${SHARED_DIR}/mgmt_iscp.yaml
```

Comment:
Suggest two whitespaces for leading `source:` and `- {@.mirrors[0]}` instead of one whitespace, that would like:
```
oc get imagecontentsourcepolicy -ojsonpath='{range .items[*].spec.repositoryDigestMirrors[*]}- mirrors:{"\n"}  - {@.mirrors[0]}{"\n"}  source: {@.source}{"\n"}{end}'
```

Reply:
i compared your command and mine, I can confirm that they are the same